source.js

```diff
@@ -0,0 +1,44 @@
+function buildTable(start, days) {
+    let table = document.getElementsByTagName("table")[0];
+    let curDay = 1;
+    for (let i = 1; i <= 6; i++) {
```

Comment:
You should receive the amount of weeks as a parameter somehow and not hard coded number.